Documentation/CONFORMANCE_TESTS.md

```diff
@@ -0,0 +1,61 @@
+# Swift Protobuf Conformance Tester
+
+**Welcome to Swift Protobuf!**
+
+Apple's Swift programming language is a perfect complement to Google's Protocol
+Buffer serialization technology.  They both emphasize high performance
+and programmer safety.
+
+Google's protobuf project includes a conformance checking host program that generates a large number of test cases, feeds them to a test program that executes those test cases, then verifies the results.  This project provides the test program for Swift Protobuf that works with Google's conformance checking host to verify that Swift Protobuf is completely compatible with other implementations of Google's protobuf specification.
```

Comment:
Looks like this file is a mix of 80c wrapping unwrapped.  wrap it all so diffs are more useful?